### PR TITLE
Manage poi_type

### PR DESCRIPTION
**Pull request by TeXitoi**
- [x] read the json configuration file
 - [x] insert the poi_type in ES
 - [x] output the poi_type in bragi

output:

```json
{
  "features": [
    {
      "properties": {
        "geocoding": {
          "poi_types": [
            {
              "name": "Jardin",
              "id": "leisure:garden"
            }
          ],
          "administrative_regions": [
            {
              "coord": {
                "lon": 2.4313619,
                "lat": 48.8011197
              },
              "weight": 0,
              "zip_codes": [
                "94700"
              ],
              "name": "Maisons-Alfort",
              "label": "Maisons-Alfort (94700)",
              "level": 8,
              "insee": "94046",
              "id": "admin:fr:94046"
            }
          ],
          "citycode": "94046",
          "city": "Maisons-Alfort",
          "postcode": "94700",
          "name": "Jardin botanique",
          "label": "Jardin botanique (Maisons-Alfort)",
          "type": "poi",
          "id": "poi:osm:way:270107085"
        }
      },
      "geometry": {
        "type": "Point",
        "coordinates": [
          2.424538,
          48.8127338
        ]
      },
      "type": "Feature"
    }
  ],
  "geocoding": {
    "query": "",
    "version": "0.1.0"
  },
  "type": "FeatureCollection"
}
```

### CODE REVIEW

File: libs/mimir/src/objects.rs [hunk 1]
```diff
@@ -28,8 +28,508 @@
...
+use serde;
+use std::rc::Rc;
+use std::cell::Cell;
+use serde::ser::Serializer;
+use std::cmp::Ordering;
 
-#[cfg(feature = "serde_codegen")]
-include!(concat!(env!("OUT_DIR"), "/objects.rs"));
+// Note: this file is needed to use serde in rust stable
+// cf mimirsbrunn/build.rs for explanations
```
Comment (antoine-de) [inline libs/mimir/src/objects.rs]: I don't understand this comment

File: libs/mimir/src/objects.rs [hunk 2]
```diff
@@ -28,8 +28,508 @@
...
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct Poi {
+    pub id: String,
+    pub label: String,
+    pub name: String,
+    pub coord: Coord,
+    pub administrative_regions: Vec<Rc<Admin>>,
+    pub weight: u32,
+    pub zip_codes: Vec<String>,
+    pub poi_type: PoiType,
```
Comment (TeXitoi) [inline libs/mimir/src/objects.rs]: For the moment yes

File: src/osm_reader/poi.rs [hunk 3]
```diff
@@ -31,59 +31,157 @@
...
 fn parse_poi(osmobj: &osmpbfreader::OsmObj,
              obj_map: &BTreeMap<osmpbfreader::OsmId, osmpbfreader::OsmObj>,
+             matcher: &PoiConfig,
              admins_geofinder: &AdminGeoFinder,
              city_level: u32)
              -> Option<mimir::Poi> {
+    let poi_type = match matcher.get_poi_type(osmobj.tags()) {
+        Some(poi_type) => poi_type,
+        None => {
+            warn!("The poi {:?} has no tags even if it passes the filters",
```
Comment (TeXitoi) [inline src/osm_reader/poi.rs]: No, OsmId only implement Debug, not Display

File: json/settings.json [hunk 4]
```diff
@@ -225,6 +225,29 @@
                             "search_analyzer": "word"
                         }
                     }
+                },
+                "poi_type": {
+                    "properties": {
+                        "id": { "type": "string", "index": "no" },
+                        "name": {
```
Comment (azime) [inline json/settings.json]: label, no ?

Comment (azime) [inline json/settings.json]: can you add an example in readme file ?

Comment (TeXitoi) [inline json/settings.json]: Added bragi output in the description. Do you want ES output?

Comment (TeXitoi) [inline json/settings.json]: I put `name` because that's exactly what we have in jormungandr and the json configuration file.

**Pull request merged**